packages/manager/modules/pci/src/projects/project/project.module.js

```diff
@@ -40,6 +40,8 @@ import pciMaintenanceBanner from '../../components/pci-maintenance-banner';
 
 import projectComponent from './project.component';
 import service from './project.service';
+import usersService from './users/users.service';
+import objectStorageService from './storages/object-storage/object-storage.service';
```

Comment:
**issue(blocking):**  Wrong injection of services.

Reply:
Component can accept function as an input and call the function. The callback function can be defined in the respective modules where the service is present.